modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/restclients/ActionsRestClient.java

```diff
@@ -30,6 +30,7 @@
 import org.wso2.identity.integration.test.utils.OAuth2Constant;
 
 import java.io.IOException;
+import java.util.Arrays;
```

Comment:
Why are we adding this import ?. Can't see any diff for using this in the class.

Reply:
Fixed with the commit ef6b7a6fcbaac733a8608c40caccf849070159cb